packages/discord.js/typings/index.d.ts

```diff
@@ -3734,7 +3734,7 @@ export abstract class DataManager<K, Holds, R> extends BaseManager {
 }
 
 export abstract class CachedManager<K, Holds, R> extends DataManager<K, Holds, R> {
-  protected constructor(client: Client<true>, holds: Constructable<Holds>);
+  protected constructor(client: Client<true>, holds: Constructable<Holds>, iterable: Iterable<Holds>);
```

Comment:
```suggestion
  protected constructor(client: Client<true>, holds: Constructable<Holds>, iterable?: Iterable<Holds>);
```
as it is not required

Reply:
Good catch, thank you!